lib/validates_zipcode/cldr_regex_collection.rb

```diff
@@ -162,7 +162,107 @@ module CldrRegexpCollection
       TC: /\ATKCA 1ZZ\z/,
       WF: /\A986\d{2}\z/,
       XK: /\A\d{5}\z/,
-      YT: /\A976\d{2}\z/
+      YT: /\A976\d{2}\z/,
+      AE: /\A([a-zA-Z\d\s]){3,}\z/,
```

Comment:
Hey @neerajkumar ! Thanks for the PR. I'm just having a look at the new additions and I have a few questions: 

* What are your sources for the new formats? I'd like to be sure we add only valid regexs.
* I see a lot of `{3,}` in the new regexs: these zipcodes have only a minimum length, but no maximum. Is that correct for all those countries, or is it just an approximation?

Thanks again!